server/templates/staff/student/assignment.timeline.html

```diff
@@ -0,0 +1,94 @@
+{% extends "staff/base.html" %}
+{% import 'staff/_formhelpers.html' as forms with context %}
+{% import 'staff/_helpers.html' as helpers %}
+
+{% block title %} {{ assignment.display_name }} Timeline {% endblock %}
+
+{% block main %}
+  <section class="content-header">
+      <h1>
+        {{ student.identifier }}'s {{ assignment.display_name }} Timeline
+      </h1>
+      <ol class="breadcrumb">
+        <li><a href="{{ url_for("admin.course", cid=current_course.id) }}">
+            <i class="fa fa-university"></i> {{ current_course.offering }}
+        </a></li>
+        <li><a href="{{ url_for('.course_assignments', cid=current_course.id) }}">
+          <i class="fa fa-list"></i> Assignments</a>
+        </li>
+        <li> <a href="{{ url_for('.assignment', cid=current_course.id, aid=assignment.id) }}"><i class="fa fa-book"></i> {{ assignment.display_name }} </a></li>
+        <li> <a href="{{ url_for('.student_view', cid=current_course.id, email=student.email) }}"><i class="fa fa-user"></i> {{ student.identifier }} </a></li>
+        <li class="active"><a href="#"><i class="fa fa-inbox"></i> History </a></li>
+      </ol>
+  </section>
+
+  <section class="content">
+    {% include 'alerts.html' %}
+
+      <div class="row">
+          <div class="col-sm-12" style="display:none">
+              {{ history }}
+          </div>
+          {% for submitter in submitters %}
+          <div class="col-md-5">
+            <!-- Default box -->
+                  <ul class="timeline">
+                      <!-- timeline time label -->
+                      <li class="time-label">
+                          <span class="bg-red">
+                              {{ submitter }} ( {{ submitters[submitter] }} backups)
+                          </span>
+                      </li>
+                      <!-- /.timeline-label -->
+                      {% for event in timeline %}
+                      <!-- timeline item -->
+                      <li style="{% if event['backup'].submitter.email == submitter %} visibility: hidden;{% endif %}" >
```

Comment:
Should just have a CSS util class that does this instead of specifying a style inline